File: stripe/src/test/java/com/stripe/android/StripeErrorFixtures.java
package com.stripe.android;

public class StripeErrorFixtures {
    public final static StripeError INVALID_REQUEST_ERROR = new StripeError(
            "invalid_request_error",
            "This payment method (bancontact) is not activated for your account.",
            "payment_method_unactivated",
            "type",
            "",
            "");
}

File: stripe/src/test/java/com/stripe/android/StripeErrorFixtures.kt
package com.stripe.android

internal object StripeErrorFixtures {
    @JvmField
    val INVALID_REQUEST_ERROR = StripeError(
        "invalid_request_error",
        "This payment method (bancontact) is not activated for your account.",
        "payment_method_unactivated",
        "type",
        "",
        ""
    )
}
